whitelabelco/sofo-foods
# Cp/feat/new product

### Description 💻

Resolves: #174 

### Implementation 💡

Client wants Cheese as a product category

Note: images for Cheese/Dairy have not been shared by the client, temporarily using the existing Dairy asset for both

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Products"
- confirm "Cheese" appears in sidebar and image grid
- click "Cheese"
- confirm routes to /cheese and cheese page displays

### Screenshots/Demos 📷

### Checklist ✅
